roles/edpm_libvirt/molecule/default/verify.yml

```diff
@@ -20,8 +20,23 @@
         - "/etc/libvirt"
     - name: ensure firewall is configured
       ansible.builtin.include_tasks: "test-helpers/verify_firewall.yaml"
-    - name: ensure ceph secret is configured
-      ansible.builtin.include_tasks: "test-helpers/verify_ceph_secret.yaml"
+
+    - name: validate tls with virt-pki-validate
+      become: true
+      ansible.builtin.shell: "virt-pki-validate"
+      register: virt_pki_validate
+    - name: Assert that virt-pki-validate returns no errors
+      assert:
+        that:
+          - "virt_pki_validate.rc == 0"
+
+    - name: ensure we can connect to libvirt with virsh via tls
+      # Note we need become because the client cert is owned by root
+      become: true
+      ansible.builtin.command: "virsh -c qemu+tls://compute-1/system version"
```

Comment:
this proably needs to be 

ansible.builtin.command: "virsh -c qemu+tls://{{ inventory_hostname }}/system version"

but ill update that if/when it fails in ci